PowerEditor/src/ScintillaComponent/FindReplaceDlg.h

```diff
@@ -124,7 +124,7 @@ friend class FindReplaceDlg;
 	void addSearchLine(const TCHAR *searchName);
 	void addFileNameTitle(const TCHAR * fileName);
 	void addFileHitCount(int count);
-	void addSearchHitCount(int count, int countSearched, bool isMatchLines, bool searchedEntireNotSelection);
+	void addSearchHitCountAndOptionsInfo(int count, int countSearched, bool searchedEntireNotSelection, const FindOption *pFindOpt);

```

Comment:
Please rename `addSearchHitCountAndOptionsInfo` to `addSearchResultInfo`.

Reply:
Done.